client/protocols/guidance.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols.common import models
+from client.utils import auth
+from urllib.request import urlopen
+import json
+import os
+import sys
+
+TGSERVER = "http://tg-server.app.cs61a.org/"
```

Comment:
I thought this was going to be in the .ok_guidance json file. But at this point I can't think of a reason to want it in the file. So we can leave this as is for now and I will not include a `'url'` key in the json.